src/docs/sphinx/developerGuide/Contributing/IntegratedTests.rst

```diff
@@ -395,227 +450,76 @@ They use a Python 3.x syntax, and have a set of ATS-related methods loaded into
-* steps: A tuple containing the test steps (minimum length = 1)
+and registers a unique test case with the `TestDeck` method, which accepts the following arguments:
 
+* name : The name of the test
+* description : A brief description of the test
+* partitions : A list of partition schemes to be tested
+* restart_step : The cycle number where GEOS should test its restart capability
+* check_step : The cycle number where GEOS should evaluate output files
+* restartcheck_params : Parameters to forward to the restart check (tolerance, etc.)
+* curvecheck_params: Parameters to forward to the curve check (tolerance, etc.)
 
-Test steps are run sequentially, and are created with the `geos` method.
-If a given test step fails, then it will produce an error and any subsequent steps will be canceled.
-This method accepts the following keyword arguments:
 
-* deck : The name of the input xml file.
-* np : The number of parallel processes required to run the step.
-* ngpu : The number of GPU devices required to run the step.  Note: this argument is typically ignored for geos builds/machines that are not configured to use GPU's.  In addition, we typically expect that np=ngpu.
-* x_partitions : The number of partitions to use along the x-dimension
-* y_partitions : The number of partitions to use along the y-dimension
-* z_partitions : The number of partitions to use along the z-dimension
-* name : The header to use for output file names
-* restartcheck_params : (optional) If this value is defined, run a restart check with these parameters (specified as a dictionary).
-* curvecheck_params : (optional) If this value is defined, run a curve check with these parameters (specified as a dictionary).
-* restart_file : (optional) The name of a restart file to resume from.  To use this option, there must be a previous step that produces the selected restart file.
-* baseline_pattern : (optional) The regex for the baseline files to use (required if the name of the step differs from the baseline)
-* allow_rebaseline : A flag that indicates whether this step can be rebaselined.  This is typically only true for the first step in a test case.
-
-
-Note that a given *.ats* file can create any number of tests and link to any number of input xml files.
-For any given test step, we expect that at least one restart or curve check be defined.
+.. note::
+  An *.ats* file can create any number of tests and link to any number of input xml files.
+  For any given test step, we expect that at least one restart or curve check be defined.
 
 
 Creating a New Test Directory
 -------------------------------
 
-To add a new set of tests, create a new folder in the `integratedTests/tests/allTests*` directory.
+To add a new set of tests, create a new folder under the `GEOS/inputFiles` directory.
 This folder needs to include at least one *.ats* file to be included in the integrated tests.
 Using the sedov example, after creating *sedov.ats* the directory should look like

```

Comment:
at some point it would be nice to enforce the `smoke` / `benchmark` nomenclature but I guess that we are not fully there yet.